src/common/options/global.yaml.in

```diff
@@ -6682,6 +6682,30 @@ options:
   desc: A configurable number for stalled read warning to be appeared if number of stalled read occurence pass `bdev_stalled_read_warn_threshold` in `bdev_stalled_read_warn_lifetime` seconds
   default: 1
   with_legacy: true
+- name: bdev_discard_max_bytes
+  type: size
+  level: advanced
+  desc: Discard queue size in bytes that triggers health warning
+  long_desc: This parameter sets a threshold for the discard queue size (in bytes), triggering a health warning when the queue exceeds the specified limit. This is particularly useful for devices with slow discard operations, as a large backlog in the queue can block disk space that is marked as free but not yet available for allocation, impacting system performance and storage efficiency. `bdev_async_discard_max_pending` is measured in items, not bytes, so its value does not directly correspond to the discard queue size in bytes
+  default: 100_K
```

Comment:
This should be orders of magnitude higher. E.g. imagine single 64Mb file release in BlueFS - this will immediately trigger the alert with your defaults.
Not to mention that such a tiny value would definitely not "block disk space that is marked as free but not yet available for allocation, impacting system performance and storage efficiency".
Generally I'm still sceptical about the need for that parameter but if it exists the threshold should be at hundreds of GBs. Or even better it should be measured in percents of total (or free) disk space, e.g. 10% of free space.